features/tbl-cell-props.feature

```diff
@@ -4,6 +4,29 @@ Feature: Get and set table cell properties
   I need a way to get and set the properties of a table cell
 
 
+  @wip
+  Scenario Outline: Get cell vertical alignment
+    Given a table cell having a vertical alignment of <vAlign-setting>
+     Then the reported vertical alignment of the cell is <reported-vAlign>
+
+    Examples: Table cell vertical alignment settings
+      | vAlign-setting        | reported-vAlign |
+      | no explicit setting   | None            |
+      | bottom                | bottom          |
+
```

Comment:
On these I've arrived at the convention of using the member name (without its enumeration "prefix" instead of a descriptive name. So "BOTTOM" in this case instead of "bottom". It has the added benefit that once you've ruled out one of the exceptional terms like "None" or "no explicit setting" the rest can be resolved in the steps code using e.g. `getattr(WD_ALIGN_VERTICAL, member)`, so you don't have to spell everything out in a dict.